invenio_jobs/services/services.py

```diff
@@ -231,8 +244,9 @@ def read(self, identity, job_id, run_id):
         """Retrieve a run."""
         self.require_permission(identity, "read")
         run = get_run(job_id=job_id, run_id=run_id)
-
-        return self.result_item(self, identity, run, links_tpl=self.links_item_tpl)
+        run_dict = run.dump()
+        run_record = AttrDict(run_dict)
```

Comment:
Question: I have seen this around and I am not sure I understand the purpose, why do we use AttrDict? and why do we need run_record?

Reply:
this a record-like implementation of a run object. It is simply done to imitate api.py layer of a run, which is missing here in this module.